docs/designs/client_tool.md

```diff
@@ -59,11 +72,15 @@ this command-line client tool.
     elasticdl zoo push a_docker_registry/bright/elasticdl-wnd:1.0
     ```
 
+    If you want to execute the job in Minikue, the `push` step
```

Comment:
Minikue -> Minikube

execute the job -> execute the job locally

Reply:
Good catch, updated.